### DISCUSSION TITLE
.NET 8 Blazor render mode heads up

### DISCUSSION THREAD

**Discussion by imkindaprogrammermyself** [vikramlearning/blazorbootstrap#535] [General]
Setting `<Routes @rendermode="@InteractiveServer" />` in `App.razor` will cause `/Account/Login` `/Account/Register` pages to repeatedly refresh itself. Removing `@rendermode="@InteractiveServer"` from Route will fix it though it will also affect Toasts and Preloads making it to not show. Setting `@rendermode="@InteractiveServer"` like `<Toasts @rendermode="InteractiveServer" class="p-3" Placement="ToastsPlacement.BottomLeft" />` on the component itself will also fix that issue. 

https://stackoverflow.com/questions/77710968/login-page-of-blazor-8-is-refreshing-in-a-loop

**Comment by gvreddy04** [comment]
@imkindaprogrammermyself Thank you for the heads up.
